licenses.yaml

```diff
@@ -1668,1285 +1604,1459 @@ libraries:
+module: java-core
+license_name: Apache License version 2.0
+version: 1.22.0
+libraries:
+  - com.google.oauth-client: google-oauth-client
+
+---
+
+name: JavaBeans Activation Framework API
+license_category: binary
+module: java-core
+license_name: CDDL 1.1
+version: 1.2.0
+copyright: Oracle and/or its affiliates
+license_file_path: licenses/bin/javax.activation-api.CDDL11
+libraries:
+  - javax.activation: javax.activation-api
 
 ---
 
-name: react-popper
+name: Apache Kafka
 license_category: binary
-module: web-console
+module: extensions/kafka-extraction-namespace
+license_name: Apache License version 2.0
+version: 0.8.2.1
+libraries:
+  - org.apache.kafka: kafka_2.10
+  - org.apache.kafka: kafka-clients
+
+---
+
+name: ZooKeeper Client
+license_category: binary
+module: extensions/kafka-extraction-namespace
+license_name: Apache License version 2.0
+version: 0.3
+libraries:
+  - com.101tec: zkclient
+
+---
+
+name: Metrics Core Library
+license_category: binary
+module: extensions/kafka-extraction-namespace
+license_name: Apache License version 2.0
+version: 2.2.0
+libraries:
+  - com.yammer.metrics: metrics-core
+
+---
+
+name: JOpt Simple
+license_category: binary
+module: extensions/kafka-extraction-namespace
 license_name: MIT License
-version: 1.3.3
-copyright: Travis Arnold
-license_file_path: licenses/bin/react-popper.MIT
+copyright: Paul R. Holser, Jr.
+version: 3.2
+license_file_path: licenses/bin/jopt-simple.MIT
+libraries:
+  - net.sf.jopt-simple: jopt-simple
+
+---
+
+name: Scala Library
+license_category: binary
+module: extensions/kafka-extraction-namespace
+license_name: BSD-3-Clause License
+copyright: EPFL, Lightbend Inc.
+version: 2.10.4
+license_file_path: licenses/bin/scala-lang.BSD3
+libraries:
+  - org.scala-lang: scala-library
 
 ---
 
-name: react-router-dom
+name: "@babel/polyfill"
 license_category: binary
 module: web-console
 license_name: MIT License
-version: 5.0.0
-copyright: React Training
-license_file_path: licenses/bin/react-router-dom.MIT
+copyright: Sebastian McKenzie
+version: 7.4.4
+license_file_path: licenses/bin/@babel-polyfill.MIT
 
 ---
 
-name: react-router
+name: "@babel/runtime"
 license_category: binary
 module: web-console
 license_name: MIT License
-version: 5.0.0
-copyright: React Training
-license_file_path: licenses/bin/react-router.MIT
+copyright: Sebastian McKenzie
+version: 7.5.4
+license_file_path: licenses/bin/@babel-runtime.MIT
+
+---
+
+name: "@blueprintjs/core"
+license_category: binary
+module: web-console
+license_name: Apache License version 2.0
+copyright: Palantir Technologies
+version: 3.17.1
 
 ---
 
-name: react-table
+name: "@blueprintjs/icons"
 license_category: binary
 module: web-console
-license_name: MIT License
-version: 6.8.6
-copyright: Tanner Linsley
-license_file_path: licenses/bin/react-table.MIT
+license_name: Apache License version 2.0
+copyright: Palantir Technologies
+version: 3.9.1
 
 ---
 
-name: react
+name: "@types/dom4"
```

Comment:
Copyright is missing for this dependency. [Its license](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE) says

> Copyrights are respective of each contributor listed at the beginning of each definition file.
>
> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
>
> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

I guess the copyright holder would be Adi Dahiya and Gilad Gray based on https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/dom4/index.d.ts#L3?